pkgs/by-name/li/libxcb/package.nix

```diff
@@ -0,0 +1,96 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkg-config,
+  python3,
+  libxau,
+  libxdmcp,
+  xcb-proto,
+  windows,
+  writeScript,
+  testers,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libxcb";
+  version = "1.17.0";
+
+  outputs = [
+    "out"
+    "dev"
+    "man"
+    "doc"
+  ];
+
+  src = fetchurl {
+    url = "mirror://xorg/individual/lib/libxcb-${finalAttrs.version}.tar.xz";
+    hash = "sha256-WZ6/mZZxD+pxYi5uGE86itW0PQ5fqMTkBxI8iKWabVU=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    pkg-config
+    python3
+  ];
+
+  buildInputs = [
+    libxau
+    libxdmcp
+    xcb-proto
+  ];
+  propagatedBuildInputs = lib.optional stdenv.hostPlatform.isMinGW windows.mingw_w64_pthreads;
+
+  # $dev/include/xcb/xcb.h includes pthread.h
```

Comment:
This comment is now in the wrong place. It should be attached to propagatedBuildInputs.

Reply:
not sure how that happened, thanks for catching it